proto-public/pbmesh/v2beta1/mesh_configuration.proto

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+syntax = "proto3";
+
+package hashicorp.consul.mesh.v2beta1;
+
+message MeshConfiguration {}
```

Comment:
The linter is going to complain about this lack of newline

Reply:
Fixed!